hpat/hiframes/pd_series_ext.py

```diff
@@ -549,14 +549,14 @@ def resolve_dropna(self, ary, args, kws):
     #         out = out.copy(dtype=types.float64)
     #     return signature(out, *args)
 
-    @bound_function("series.pct_change")
-    def resolve_pct_change(self, ary, args, kws):
-        # TODO: support default period argument
-        out = ary
-        # integers are converted to float64 to store NaN
-        if isinstance(ary.dtype, types.Integer):
-            out = out.copy(dtype=types.float64)
-        return signature(out, *args)
+    # @bound_function("series.pct_change")
```

Comment:
You are likely to break the multiprocessing parallelism by disabling "old style" implementation.

Reply:
it is not implementation - it is typing stage. Some time it is ok to disable